cime_config/testdefs/testlist_clm.xml

```diff
@@ -4212,7 +4234,7 @@
       <option name="comment">Initialization test for ne30pg3 with "Large" layout</option>
     </options>
   </test>
-  <test name="SMS_Ln1_P43x1" grid="ne30pg3_ne30pg3_mt232" compset="I2000Clm45Sp" testmods="clm-run_self_tests">
+  <test name="SMS_Ln1_P43x1" grid="ne30pg3_ne30pg3_mt232" compset="I2000Clm45Sp" testmods="clm-for_testing_fastsetup_bypassrun">
```

Comment:
@ekluzek I didn't catch this in my review, but such renames are annoying during the b4b-dev merge. I will make some softlinks in the baselines so these don't throw errors.

Reply:
Oh, sorry, I see now these aren't on aux_clm so they won't be affected. Never mind!